services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -2,6 +2,7 @@
 from flask_restplus import Resource, reqparse, inputs
```

Comment:
In this file, I mainly just adjusted indentation to reduce nesting so its easier to read and updated some comments.